src/VisualStudio/Core/Def/Implementation/ProjectSystem/Interop/IVsSolution7.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.Interop
+{
+    [ComImport]
+    [Guid("D32B0C42-8AEE-4772-B5C3-04565CDA5A47")]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    public interface IVsSolution7
```

Comment:
And yet it didn't.  Apparently not having a `PublicAPI.Unshipped.txt` is enough to defeat the analyzer entirely...

https://github.com/dotnet/roslyn-analyzers/blob/master/src/Roslyn.Diagnostics.Analyzers/Core/DeclarePublicAPIAnalyzer.cs#L149